services/app-web/src/common-code/healthPlanFormDataType/healthPlanFormData.ts

```diff
@@ -228,16 +251,65 @@ const generateRateName = (
     return rateName
 }
 
+const removeRateRelatedDocuments = (
+    documents: SubmissionDocument[]
+): SubmissionDocument[] => {
+    const noRateDocs: SubmissionDocument[] = []
+    documents.forEach((document) => {
+        const ratesRelatedDocIndex =
+            document.documentCategories.indexOf('RATES_RELATED') === -1
```

Comment:
Is converting to undefined serving any purpose here?  I think you could just get the index and check below if it's -1.

Reply:
Good catch! It was until, I found a bug with the code. Now it's unnecessary left over code.